### PR TITLE
add colors

### PR DESCRIPTION
**Pull request by daniela-93**

### CODE REVIEW

**Review by Sha-94 — DISMISSED** [review]

**Review by frecklescherokee — DISMISSED** [review]
colors are dope

**Review by Aeromero — DISMISSED** [review]

File: assets/css/style.css [hunk 1]
```diff
@@ -1,3 +1,9 @@
 body{
     background-color: aqua;
+}
+:root{
+--primary-color: purple;
+--secondary-color: gold;
+--tertiary-color: grey;
```
Comment (Sha-94) [inline assets/css/style.css]: use a different color

**Review by Sha-94 — COMMENTED** [review]
LEft a comment

**Review by Sha-94 — APPROVED** [review]

**Review by Aeromero — APPROVED** [review]

**Review by frecklescherokee — APPROVED** [review]
word

**Review by alyssafadai — COMMENTED** [review]
Yas queen good job!

**Pull request merged**